pandas/core/indexes/base.py

```diff
@@ -7227,7 +7256,9 @@ def _maybe_disable_logical_methods(self, opname: str_t) -> None:
             raise TypeError(f"cannot perform {opname} with {type(self).__name__}")
 
     @Appender(IndexOpsMixin.argmin.__doc__)
-    def argmin(self, axis=None, skipna: bool = True, *args, **kwargs) -> int:
+    def argmin(
+        self, axis: int | None = None, skipna: bool = True, *args, **kwargs
```

Comment:
Might be slighty better to use `pandas._typing.AxisInt | None` (in the case we use Literals in the future for axis)

Reply:
some for the next three methods